tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -4723,9 +4738,7 @@ <h2>Show similarity to selected datapoint</h2>
         });
         this._requestManager.request(url, postData).then(wrapperFn)
             .catch(reason => {
-              this.exampleStatusStr = 'Request failed';
-              this.showToast_('Request failed: ' + reason);
-              this.$.spinner.hidden = true;
+              this.handleError(requestStr + ' failed: ' + reason);
```

Comment:
```ts
`Request for ${humanReadableRequestName} failed: ${reason}`
```

I thought it would be confusing to show a toast that says something like `Model inference failed: foobarbaz`

Reply:
done